### PR TITLE
GIX-2060: Move nns fee stores

### PR DESCRIPTION
**Pull request by lmuntaner**
# Motivation

Remove the `transactionsFeesStore`.

In this PR, move the derived stores that were in `transactions-fees.store.ts` along with `transactionsFeesStore` to `main-transactoin-fee.derived.ts`. This will allow me to remove the file along the store in a following PR.

# Changes

* Move `mainTransactionFeeStore` and `mainTransactionFeeE8sStore` to  `main-transactoin-fee.derived.ts`.
* Fix imports.

# Tests

* Move the test of `mainTransactionFeeStore` and add a test for `mainTransactionFeeE8sStore`.
* Rename "transaction-fee.derived.spec" to "main-transaction-fee.derived.spec".

# Todos

- [ ] Add entry to changelog (if necessary).

Not necessary.

### CODE REVIEW

**Comment by lmuntaner** [comment]
Current dependencies on/for this PR:
* `main`
  * **PR #4301** <a href="https://app.graphite.dev/github/pr/dfinity/nns-dapp/4301?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/dfinity/nns-dapp/4301?utm_source=stack-comment).

File: frontend/src/lib/derived/main-transaction-fee.derived.ts [hunk 1]
```diff
@@ -1,13 +1,43 @@
...
+    return TokenAmount.fromE8s({ amount: icpToken.fee, token: icpToken });
   }
-  return TokenAmount.fromE8s({ amount: icpToken.fee, token: icpToken });
-});
+);
+
+/**
+ * @deprecated prefer mainTransactionFeeE8sStore to use e8s for amount of tokens instead of Number.
+ */
+export const mainTransactionFeeStore: Readable<number> = derived(
```
Comment (lmuntaner) [inline frontend/src/lib/derived/main-transaction-fee.derived.ts:21]: No changes.

File: frontend/src/lib/derived/main-transaction-fee.derived.ts [hunk 2]
```diff
@@ -1,13 +1,43 @@
...
+    const icpToken = $store[OWN_CANISTER_ID_TEXT]?.token;
+    if (isNullish(icpToken)) {
+      // This can't happen because the tokensStore always contains the NNS token.
+      throw new Error("ICP token not found");
+    }
+    return Number(icpToken.fee);
+  }
+);
+
+export const mainTransactionFeeE8sStore: Readable<bigint> = derived(
```
Comment (lmuntaner) [inline frontend/src/lib/derived/main-transaction-fee.derived.ts:33]: No changes

File: frontend/src/lib/derived/main-transaction-fee.derived.ts [hunk 3]
```diff
@@ -1,13 +1,43 @@
...
 import { TokenAmount, isNullish } from "@dfinity/utils";
-import { derived } from "svelte/store";
+import { derived, type Readable } from "svelte/store";
 
-export const mainTransactionFeeStoreAsToken = derived(tokensStore, ($store) => {
-  const icpToken = $store[OWN_CANISTER_ID_TEXT].token;
-  if (isNullish(icpToken)) {
-    // This can't happen because the tokensStore always contains the NNS token.
-    throw new Error("ICP token not found");
+export const mainTransactionFeeStoreAsToken: Readable<TokenAmount> = derived(
```
Comment (lmuntaner) [inline frontend/src/lib/derived/main-transaction-fee.derived.ts:6]: I added `Readable<TokenAmount>`.

**Comment by lmuntaner** [comment]
@dskloetd please review

File: frontend/src/tests/lib/derived/transaction-fee.derived.spec.ts [hunk 4]
```diff
@@ -11,4 +17,34 @@ describe("mainTransactionFeeStoreAsToken", () => {
     const value = get(mainTransactionFeeStoreAsToken);
     expect(value instanceof TokenAmount).toBeTruthy();
   });
+
+  it("should set ICP fee value as number", () => {
+    expect(get(mainTransactionFeeStore)).toBe(Number(NNS_TOKEN_DATA.fee));
```
Comment (dskloetd) [inline frontend/src/tests/lib/derived/transaction-fee.derived.spec.ts:22]: Shouldn't this go in a different main-specific test file?

Comment (lmuntaner) [reply frontend/src/tests/lib/derived/transaction-fee.derived.spec.ts]: I'll rename the file. It's a typo there.

**Review by dskloetd — APPROVED** [review]

File: frontend/src/tests/lib/derived/main-transaction-fee.derived.spec.ts
Comment (lmuntaner) [inline frontend/src/tests/lib/derived/main-transaction-fee.derived.spec.ts:1]: This is the same file removed "transaction-fee.derived.spec" with the tests for the new derived stores `mainTransactionFeeE8sStore` and `mainTransactionFeeStore`.

**Pull request merged**